pluma/pluma-pango.c

```diff
@@ -104,7 +104,9 @@ pluma_pango_font_description_to_css (const PangoFontDescription *desc)
   if (set & PANGO_FONT_MASK_FAMILY)
     {
       g_string_append (s, "font-family: ");
+      g_string_append (s, "\"");
       g_string_append (s, pango_font_description_get_family (desc));
+      g_string_append (s, "\"");
       g_string_append (s, "; ");
```

Comment:
As @zhuyaliang suggests, and it could actually improve the whole thing:
```suggestion
      g_string_append_printf (s, "font-family: \"%s\"; ",
                              pango_font_description_get_family (desc));
```